Dockerfile

```diff
@@ -4,6 +4,9 @@ MAINTAINER Erlang Solutions <mongoose-im@erlang-solutions.com>
 
 WORKDIR /amoc_arsenal
 
+COPY rebar.config rebar.lock ./
+RUN rebar3 compile -d
```

Comment:
this change must be reverted

Reply:
I cannot see why on earth would you not want to cache dependencies but ok whatever 🤷🏽

Reply:
you don't really cache them,  2 following lines just reset everything you do here.

Reply:
Then the two following lines are wrong 😄 
Anyway an optimisation for another day, will remove from here, I reeeeeeally don't want to get this PR any complicated.